best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
```

Comment:
Can someone define a "service class"? Are we talking about a command object? Or are we more broadly speaking of classes that are not rails framewrok related?